css/src/components/table.scss

```diff
@@ -0,0 +1,67 @@
+$table-cell-vertical-padding: $spacer-4 !default;
+$table-cell-horizontal-padding: $spacer-5 !default;
+
+$table-caption-padding: $spacer-6 !default;
+$table-caption-size: $font-size-9 !default;
+$table-caption-spacing: $letter-spacing-medium !default;
+
```

Comment:
I think for migration purposes, it will be useful if we can include a feature flag here:

```

$features-button: true !default;

@if $features-button == true {
[...]
```

Reply:
Updated